src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        SimpleDateFormat dateFormat = new SimpleDateFormat("dd.MM.yyyy");
+        Date fromDate = null;
+        try {
+            fromDate = dateFormat.parse(dateFrom);
+        } catch (ParseException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
amount of try/catches in code makes it hard to read
please introduce separate private method (like getDateFromString() or similar), move try/catch block there and reuse in main method to make it cleaner